server/swagger.json

```diff
@@ -5143,6 +5143,60 @@
         }
       }
     },
+    "LogViewerUIColumn": {
+      "description": "Contains the settings for the log viewer page UI",
+      "type": "object",
+      "required": [
+        "name",
+        "formatting",
+        "mappings"
+      ],
+      "properties": {
+        "name": {
+          "description": "Unique identifier name of the column",
+          "type": "string"
+        },
+        "formatting": {
+          "description": "Composable formatting options for the column",
+          "type": "array",
+          "items": {
+            "description":"Type and value of a formatting option",
+            "type": "object",
+            "required": ["type", "value"],
+            "properties": {
+              "type": {
+                "type": "string"
+              },
+              "value": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "mappings": {
```

Comment:
What does "mappings" mean in this context? For example, when a developer in the future wants to add another editable field to the columns, how do they know if what data should fall under "formatting" or "mappings"? @goller

Reply:
@AlirieGray I actual prefer only having one field, `formatting`.  This design comes from some desire to split those meanings.

If it is too difficult to determine which context it falls under (and I think it is too difficult) then that's a good hint it should change to just one.